apps/condo/domains/marketplace/components/MarketItem/UpdateMarketItemForm.tsx

```diff
@@ -179,7 +190,7 @@ export const UpdateMarketItemForm = ({ marketItem }) => {
 
         setSubmitLoading(false)
 
-        await router.push(`/marketplace/marketItem/${get(marketItem, 'id')}`)
+        // await router.push(`/marketplace/marketItem/${get(marketItem, 'id')}`)
```

Comment:
maybe leave this? it’s better to throw it at 404 if this screen is not ready yet, otherwise we’ll forget to uncomment it and have to do an extra pull request